Encrypted subscription channels (#2663)
nuwave/lighthouse: docs/master/subscriptions/getting-started.md

The easiest way to expire subscriptions automatically is to use the env `LIGHTHOUSE_SUBSCRIPTION_STORAGE_TTL`
to set an expiration time in seconds (e.g. `LIGHTHOUSE_SUBSCRIPTION_STORAGE_TTL=3600` to expire in one hour).

## Encrypted Subscriptions

> Lighthouse supports encrypted subscriptions for both Pusher and Echo drivers. The main goal is to ensure that all data in transit in a private-encrypted channel is encrypted. 

Reviewer: ```suggestion
Lighthouse supports encrypted subscriptions for the drivers Pusher and Echo.
The main goal is to ensure that all data in a private-encrypted channel is encrypted in transit. 
```

> To enable encrypted subscriptions, set the environment variable `LIGHTHOUSE_SUBSCRIPTION_ENCRYPTED=true`.

Reviewer: Please reference the config setting instead.

For Pusher, the shared secret can be updated in `config/broadcasting.php`:

```php
// in config/broadcasting.php
'options' => [
  'useTLS' => true,
  'encryption_master_key_base64' => 'YOUR_MASTER_KEY', // generate this with, e.g. 'openssl rand -base64 32'
],
```

> More details on how to enable encryption are defined in the [Pusher documentation](https://pusher.com/docs/channels/using_channels/encrypted-channels/).

Reviewer: ```suggestion
More details on how to enable encryption are defined in the [Pusher documentation](https://pusher.com/docs/channels/using_channels/encrypted-channels).
```

The easiest way to expire subscriptions automatically is to use the env `LIGHTHOUSE_SUBSCRIPTION_STORAGE_TTL`
to set an expiration time in seconds (e.g. `LIGHTHOUSE_SUBSCRIPTION_STORAGE_TTL=3600` to expire in one hour).

## Encrypted Subscriptions

Lighthouse supports encrypted subscriptions for the drivers Pusher and Echo.
The main goal is to ensure that all data in a private-encrypted channel is encrypted in transit. 

> To enable encrypted subscriptions, just set the `subscriptions.encrypted_channels` to `true` in `lighthouse.php`.

Reviewer: ```suggestion
To enable encrypted subscriptions, set `subscriptions.encrypted_channels` to `true` in `config/lighthouse.php`.
```
